### DISCUSSION TITLE
How to import SSH sessions from PuTTY?

### DISCUSSION THREAD

**Discussion by Duayne2001** [Eugeny/tabby#5497] [Q&A] [answered]
I may have overlooked it but is there a way to import my PuTTY sessions into Tabby using Windows?

Thanks.

**Comment by Eugeny** [comment, marked as answer]
There is no way to do this yet.

**Reply by 001101** [reply]
great...

**Reply by nacesprin** [reply]
I love that!

**Comment by josevara2020** [comment]
Dear, there will be a graphic manual with the steps to follow from putty ssh to tabby ssh.
since I tried to carry out that procedure without success. SSH works fine in putty but not in tabby?
please help.

**Comment by ajkrj** [comment]
I faced the exact same issue and I created a set of PowerShell scripts with the help of AI that can migrate PuTTY sessions to Tabby. 

They read your saved PuTTY sessions directly from the Windows Registry and convert them to Tabby's YAML format.

Repository: [https://github.com/ajkrj/putty-tabby](https://github.com/ajkrj/putty-tabby)

They work on Windows where PuTTY stores sessions in HKEY_CURRENT_USER\Software\SimonTatham\PuTTY\Sessions

You can use the output yaml files and paste it into your Tabby Terminal Config. 
If you have any existing config, there is another script on the repository which tries to merge both configs

Hope this helps others looking to migrate

**Reply by dusfor72** [reply]
Confirmed, works, thanks for the hint.
